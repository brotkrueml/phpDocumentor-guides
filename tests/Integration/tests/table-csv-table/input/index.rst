Csv table
====================

.. csv-table::
   :file: ./test.csv

test
