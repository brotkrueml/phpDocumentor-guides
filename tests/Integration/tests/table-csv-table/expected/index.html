<!DOCTYPE html>
<html lang="en">
<head>
    <title>Csv table</title>

</head>
<body>
<div class="section" id="csv-table">
    <h1>Csv table</h1>

    <table>

        <tbody>
        <tr>
            <td>col 1</td>
            <td>col 2</td>
        </tr>
        <tr>
            <td>a</td>
            <td>b</td>
        </tr>
        <tr>
            <td>c</td>
            <td>d</td>
        </tr>
        </tbody>
    </table>

    <p>test</p>
</div>

</body>
</html>
